psydac/api/fem.py

```diff
@@ -1039,18 +1128,47 @@ def allocate_matrices(self):
 
 #==============================================================================
 class DiscreteFunctional(BasicDiscrete):
+    """ Class that represents the concept of a discrete functional form.
+        This class generates the functiona form assembly method.
+
+    Parameters
+    ----------
+
+    expr : sympde.expr.expr.LinearForm
+        The symbolic functional form.
+
+    kernel_expr : sympde.expr.evaluation.KernelExpression
+        The atomic representation of the functional form.
+
+    domain_h : Geometry
+        The discretized domain
+
+    space : FemSpace
+        The discrete spaces.
```

Comment:
Typo: space -> spaces

Reply:
fixed in [9ed13cb](https://github.com/pyccel/psydac/pull/232/commits/9ed13cb7ef3cc0b2cf5ef6284c62b95ffd2874f1)